### DISCUSSION TITLE
user-friendly principles for community empowerment

### DISCUSSION THREAD

**Discussion by annyrviloria** [albertkun/21S-ASIAAM-191A#64] [Past Thinking Caps]
Not to evade the question, but as a non-cartographer (or new cartographer?) I would want to make space to learn from the community leaders/agents of change/activists who've been leading the struggle and collecting lessons along the way. I want to honor the fact that many before us have been engaged in decolonial geopolitics and that the distance between academia and the real world is not to take for granted. [NPR put out a very interesting article this week](https://www.npr.org/sections/goatsandsoda/2021/04/06/982708354/the-mapmaker-whos-saving-lives-in-panamas-pandemic) telling the story of Carlos Doviaza, an indigenous, self-taught Panamenian cartographer whose maps are being used to allow indigenous communities to advocate for themselves. One of the things I took away from his work is that he hopes to make his maps as user-friendly as possible, to make sure the indigenous communities being "benefitted" by the maps also understand them and feel in ownership of it. This lesson could be extended to our own work, by ensuring we keep the user in mind not just for the sake of it, but as a way to ground ourselves in the humanity and dignity of the communities that are trusting us. I think this can be a successful strategy to incentivize trust in the use of maps and to allow communities to integrate these tools in other aspects of their self-advocacy.

**Comment by KenNL42** [comment]
I agree with having a mindset of making a map as user-friendly as possible. This is because maps are supposed to be understandable by anybody, from anywhere around the world. Anyone can look at the map and understand what message the cartographer tries to convey. However, this is definitely a difficult challenge to the cartographer as they have to experience different community and background in order to find a common ground that anybody can understand what is the map for.

**Comment by vcta99** [comment]
Also agree with prioritizing user needs in cartography/digital spaces.  I think that, before publication/release of any project, maps should be vigorously user tested and reviewed by the communities it seeks to represent. This "approval" process would make maps more consentful and allow an opportunity for the cartographers to have their assumptions challenged.
